v2/test/env.spec.ts

```diff
@@ -344,7 +344,7 @@ describe("INJECT_LOG_CONTEXT_ENV_VAR", () => {
       },
     });
     const hello = new lambda.Function(stack, "HelloHandler", {
-      runtime: lambda.Runtime.NODEJS_12_X,
+      runtime: lambda.Runtime.NODEJS_16_X,
```

Comment:
there is 10x18 = 180 loc that can be extracted to a function here. Is it worth extracting it?
```
const app = new App();
    const stack = new Stack(app, "stack", {
      env: {
        region: "us-west-2",
      },
    });
    const hello = new lambda.Function(stack, "HelloHandler", {
      runtime: lambda.Runtime.NODEJS_16_X,
      code: lambda.Code.fromInline("test"),
      handler: "hello.handler",
    });
    ```

Reply:
I'll take a look into it.

Reply:
By definition, we will still be doing the same allocations, due to the state of the stack being changed on every test. I'd rather do this is in another PR where we restructure testing. WDYT?